common/model-views.cpp

```diff
@@ -1951,13 +1911,31 @@ namespace rs2
         return results;
     }
 
-    std::vector<stream_profile> subdevice_model::get_selected_profiles()
+    bool is_resolution_max(const std::vector<std::pair<int, int>>& res_values, int width, int height, const std::pair<int, int>& calibration_res)
+    {
+        auto size = res_values.size();
+        int max_width = (res_values[size - 1].first != calibration_res.first) ? res_values[size - 1].first : res_values[size - 2].first;
+        int max_height = (res_values[size - 1].second != calibration_res.second) ? res_values[size - 1].second : res_values[size - 2].second;
+        return width == max_width && height == max_height;
+    }
+
+    std::vector<stream_profile> subdevice_model::get_selected_profiles(bool throw_exception_flag)
     {
         std::vector<stream_profile> results;
 
         std::stringstream error_message;
         error_message << "The profile ";
 
+
+        const int d405_calibration_width = 1288;
```

Comment:
Not sure it is needed - Y16 is sufficient as it supports only a single resolution

Reply:
without the resolution written there:
![image](https://user-images.githubusercontent.com/55648466/125752991-4c4d76e0-c971-4082-95ea-a28f258f5364.png)